global/templates/pages/get-involved.html

```diff
@@ -15,46 +15,17 @@
-  <h2>Transcribing a mission</h2>
-  <p>From the smallest task to one of the biggest, transcribing a mission from scratch is a much more intensive job than correcting errors in our existing missions. However it is still non-technical, although a reasonable grasp of English and some understanding of the kind of terminology of space flight can help a lot!</p>
-  <p>The first thing to do is to to pick a mission (see <a href="https://github.com/Spacelog/Spacelog/wiki/Missions">our mission roster</a> for what's already been claimed, or talk to us if you have no preference), and download the "TEC" (air-to-ground) transcript for the relevant mission from <a href="http://www.jsc.nasa.gov/history/mission_trans/mission_transcripts.htm">NASA</a>. Then you need to grab the text of that; either open up the PDF and select all and copy out into a text editor or Word or something, or if you have it use something like pdftotext (on Debian/Ubuntu it's part of the xpdf-utils package) to pull the text out. (pdftotext generally does a better job than copying out of a PDF viewer, so if you don't have access to it, let us know and we can run it for you.)</p>
-
-  <p>Then you have to "clean" it; the text version of the PDF was created automatically by a computer, so it has some errors, so you need to go through the text version looking for those errors, and checking against the PDF to fix problems. (You can skip the first few pages, which are boilerplate we'll convert into things like the glossary later.) Log lines (a timestamp, speaker and some text) should look like this:</p>
-
-  <pre>00 00 09 32 LMP Charlie, lots of stuff out the window in staging.
-                       We're catching up and passing it now.</pre>
-
-  <p>Time stamps are days hours minutes seconds, then speakers are all caps abbreviations like "LMP" for Lunar Module Pilot, "CC" for the Capsule Communicator on the ground, and so on. The text can be on more than one line, as demonstrated above.</p>
-
-  <p>Things <em>other</em> than log lines are important as well. There are a few of these, and they should all end up indented by (at least) one tab:</p>
-
-  <pre>      Tape 1/6
-        Page 6</pre>
-
-  <p>these come at the top of every page. They're important because they help us link passages in the transcript back to the original PDF pages.</p>
+  <h2>Help transcribe a new mission</h2>
+  <p>The easiest way to contribute to Spacelog is by helping to transcribe new missions using Kallisto, our online transcription tool. Transcribing a page or two only takes a few minutes, and you can do as much or as little as you like.</p>
```

Comment:
Given Kallisto doesn't really contain any help on what to do, we should either have more detail here, or we should put something in Kallisto itself.